draft-ietf-quic-http.md

```diff
@@ -1418,13 +1413,11 @@ The entries in the following table are registered by this document.
 |----------------------------|------|-------------------------|
 | Setting Name               | Code | Specification           |
 |----------------------------|:----:|-------------------------|
-| HEADER_TABLE_SIZE          | 0x1  | {{settings-parameters}} |
```

Comment:
It strikes me as bizarre that 0x1 is omitted from this table / or that QPACK - as an extension or subordinate document - gets to define the lowest numbered setting value.